revolv/static/js/base/dashboard.js

```diff
@@ -20,6 +20,73 @@ $(document).ready(function () {
             $this.addClass("fa-rotate-180");
         }
     });
+    
+    /**
+     * This function defines what happens when the different categories are clicked in the
+     * 'my impact' section on the dashboard. The category that is clicked is made active, 
+     * and then an AJAX request is made. The view this AJAX reqest is sent to updates the
+     * preferred_categories attribute of this user.
+     * 
+     */
+    $(".category-option-container").click(function() {
+        $(this).toggleClass("active");
+        var allActiveCategories = $(".category-option-container.active");
+        var allCategories = $(".category-option-container");
+        var dict = {};
+        for (var i = 0; i < allCategories.length; i += 1) {
+            var categoryText = $(allCategories[i]).children().last().text();
```

Comment:
Can you use [.find()](https://api.jquery.com/find/) instead of grabbing the last child? If we changed the markup, then this would fail in its current state. Also, can we use the category id (perhaps as a [data](https://api.jquery.com/jquery.data/) attribute) instead of the text? In general, always better to specify ids rather than titles, because ids will never change.